components/QuizCardPage/QuizPageForm/index.test.js

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from "@testing-library/react";
+import QuizCard from "../../../pages/QuizCard";
+
+describe("StyledForm component", () => {
```

Comment:
Your test describes the `StyledForm`, but you are rendering `QuizCard` inside the test, which is a page. This is confusing.